tests/src/test/scala/cats/tests/XorTests.scala

```diff
@@ -46,6 +47,12 @@ class XorTests extends CatsSuite {
     checkAll("Eq[ListWrapper[String] Xor ListWrapper[Int]]", SerializableTests.serializable(Eq[ListWrapper[String] Xor ListWrapper[Int]]))
   }
 
+  {
+    implicit val L = ListWrapper.semigroup[String]
```

Comment:
and, yeah, coming from Haskell it feels like these instances are just flying around and I have no idea what's happening :) For example, I didn't know you could introduce a local instance by wrapping something in curly braces.